docs/content/latest/manage/upgrade-deployment.md

```diff
@@ -110,7 +114,7 @@ For a successful YSQL upgrade, a message will be displayed as follows:
 YSQL successfully upgraded to the latest version
 ```
 
-In certain scenarios, YSQL upgrade can take longer than 60 seconds, which is the default timeout value for `yb-admin`. To account for that, run the command with a higher timeout value:
+In certain scenarios, a YSQL upgrade can take longer than 60 seconds, which is the default timeout value for `yb-admin`. To account for that, run the command with a higher timeout value:
```

Comment:
What happens if yb-admin times out while the upgrade is running?

Reply:
yeah, that is a good suggestions..

Reply:
Thank you @polarweasel and @sushantrmishra. Let me add it to the `yb-admin` page as well.